packages/manager/modules/pci/src/projects/project/instances/instance/instance.html

```diff
@@ -529,12 +529,19 @@
                             ></span>
                         </oui-action-menu-item>
                         <oui-action-menu-item
-                            data-disabled="$ctrl.disablePrivateNetworks"
+                            data-disabled="!!$ctrl.instance.privateNetworks.length"
                             data-on-click="$ctrl.attachPrivateNetwork()"
                             ><span
                                 data-translate="pci_projects_project_instances_instance_private_networks_action_attach_label"
                             ></span>
                         </oui-action-menu-item>
+                        <oui-action-menu-item
+                            data-disabled="!$ctrl.instance.privateNetworks.length"
+                            data-on-click="$ctrl.detachPrivateNetwork()"
```

Comment:
i guess better to have event prefix naming convention.
```suggestion
                            data-on-click="$ctrl.onDetachPrivateNetworkClick()"
```

Reply:
Done.